joplin/base/tests/test_information_page.py

```diff
@@ -0,0 +1,18 @@
+from base.models import InformationPageContact
+from base.factories.information_page import InformationPageContactFactory, InformationPageFactory
+import pytest
+
+
+@pytest.mark.django_db
```

Comment:
this decorator means 'i need the db for this test' 
Thats it! no setup, teardown code required